spot_driver/spot_driver/spot_ros2.py

```diff
@@ -457,6 +458,9 @@ def __init__(self, parameter_list: Optional[typing.List[Parameter]] = None, **kw
 
         self.create_subscription(Twist, "cmd_vel", self.cmd_velocity_callback, 1, callback_group=self.group)
         self.create_subscription(Pose, "body_pose", self.body_pose_callback, 1, callback_group=self.group)
+        self.create_subscription(
+            JointState, "arm_joint_commands", self.arm_joint_cmd_callback, 100, callback_group=self.group
```

Comment:
Just out of curiosity is there a standard topic name for this? Something analogous to `cmd_vel`

Reply:
Not that I know of.... for specifically only the arm joints. For all joints `/joint_command` is pretty common i think in ros uses but specifically only for arm, I am not sure